errai-cdi/jboss/src/main/java/org/jboss/errai/cdi/server/gwt/JBossLauncher.java

```diff
@@ -45,13 +49,17 @@
   public ServletContainer start(TreeLogger treeLogger, int port, File appRootDir) throws BindException, Exception {
     logger = new StackTreeLogger(treeLogger);
 
-    logger.branch(Type.INFO, "Starting launcher...");
+    logger.branch(Type.INFO, "Server launcher starting..");
 
     // Get properties
-    final String DEBUG_PORT = System.getProperty(JBOSS_DEBUG_PORT_PROPERTY, "8001");
-    final String TEMPLATE_CONFIG_FILE = System.getProperty(TEMPLATE_CONFIG_FILE_PROPERTY, "standalone-full.xml");
+    final String DEBUG_PORT 		     = System.getProperty(JBOSS_DEBUG_PORT_PROPERTY, "8001");
+    final String TEMPLATE_CONFIG_FILE    = System.getProperty(TEMPLATE_CONFIG_FILE_PROPERTY, "standalone-full.xml");
     final String CLASS_HIDING_JAVA_AGENT = System.getProperty(CLASS_HIDING_JAVA_AGENT_PROPERTY);
     String JAVA_OPTS = System.getProperty(JBOSS_JAVA_OPTS_PROPERTY, "");
+    
+    // Added new properties to allow overriding of default ones.
+    final String HTTP_REMOTING_ADDRESS      =System.getProperty(JBOSS_HTTP_REMOTING_ADDRESS, "http-remoting://localhost:9990");
```

Comment:
Please format this line and the one below so that there is a single space on either side of the equals symbol.